samples/apps/bookinfo/cluster_extension.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+#
```

Comment:
Not really liking the concept of polluting bookinfo root. If possible move to dedicated folder. Also, call it demo_cluster_ext.sh

Reply:
Or just _bookinfo_vm_svc.sh

Reply:
Sure - how about cluster_ext folder ? We need some way to help automate running
bookinfo on a VM.

Reply:
cluster_ext or scripts.. your call.

Reply:
Used clusterex for now ( since it's not clear if 'ext' or 'exp' will be used :-)